code/modules/projectiles/guns/special/medbeam.dm

```diff
@@ -123,6 +123,14 @@
 				qdel(dummy)
 				return FALSE
 		for(var/obj/effect/ebeam/medical/B in next_step)// Don't cross the str-beams!
+			if(QDELETED(current_beam))
+				break //We shouldn't be processing anymore.
+			if(QDELETED(B))
+				continue
+			if(!B.owner)
+				message_admins("BEAM WITHOUT AN OWNER. SOMETHING IS TERRIBLY WRONG, YELL AT A CODER")
```

Comment:
do we need a message_admins for this? someday i should make a better runtime viewer